src/test/java/com/epam/atm/selenium/LoginPOTest.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.atm.selenium;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pageobject.HomePage;
+import pageobject.LoginPage;
+
+public class LoginPOTest extends BaseTest {
+    private static final String LOGIN = "yauhen_valodzin@epam.com.qa";
+    private static final String PASSWORD = "Ceakt_1234";
+    private static final String TITLE_HOME_PAGE = "Recently Viewed | Applications | Salesforce";
+
+    @Test(description = "Check Login to SalesForce is working")
+    public void loginToSF() {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.openURL();
+        System.out.println("Check Username field is displayed");
```

Comment:
move System.out.println("Check Username field is displayed"); to method loginPage.checkUsernameIsDisplayed()

same with other displayed messages

Reply:
done